Formula/cortex.rb

```diff
@@ -5,6 +5,11 @@ class Cortex < Formula
   sha256 "eba55bdb420d91da7c6cfc056172cb613298d3429439c175f044d58da4e0ecf7"
   license "Apache-2.0"
 
+  livecheck do
+    url "https://github.com/cortexproject/cortex/releases"
```

Comment:
```suggestion
    url "https://github.com/cortexproject/cortex/releases/latest"
```

This should be checking the "latest" release.

As explained in the past, we almost never check the first page of the GitHub releases, since prerelease versions may push the latest stable version onto the second page (breaking the check). When the GitHub repository doesn't have a "latest" release (or it's not correct), we simply check the Git tags instead.

Reply:
my bad, not sure how i missed that.